File: class/student.php
<?php

    include_once 'account.php';

    class Student
    {
        private const student_table = 'Student';

        private PDO $connect;

        public function __construct (PDO $connect)
        {
            $this->connect = $connect;
        }

        public function insert ($sql_data, $part_of_sql)
        {
            $this->_insert($part_of_sql, $sql_data);
        }

        private function _insert ($part_of_sql, $sql_data)
        {
            $sql_query =
                'INSERT INTO 
                    ' . self::student_table . ' 
                (
                    ID_Student, Student_Name, DoB_Student, ID_Class, 
                    ID_Card_Number, Phone_Number_Student, Address_Student
                ) 
                VALUES 
                    ' . $part_of_sql . '
                ON DUPLICATE KEY UPDATE ID_Student = ID_Student';

            try {
                $stmt = $this->connect->prepare($sql_query);
                $stmt->execute($sql_data);

            } catch (PDOException $error) {
                throw $error;
            }
        }
    }

File: worker/student.py
class Student:
    def __init__(
        self,
        ID_Student,
        Student_Name,
        DoB,
        ID_Class,
        ID_Card_Number=None,
        Phone_Number_Student=None,
        Address_Student=None,
    ) -> None:
        self.ID_Student = ID_Student
        self.Student_Name = Student_Name
        # d/m/y -> y/m/d
        self.DoB = "/".join(DoB.split("/")[::-1])
        self.ID_Class = ID_Class
        self.ID_Card_Number = ID_Card_Number
        self.Phone_Number_Student = Phone_Number_Student
        self.Address_Student = Address_Student

    def __eq__(self, other: object) -> bool:
        if isinstance(other, Student):
            return self.ID_Student == other.ID_Student

        return False

    def __ne__(self, other: object) -> bool:
        return not self.__eq__(other)

    def __str__(self) -> str:
        return (
            "("
            + ", ".join([self.ID_Student, self.Student_Name, self.ID_Class, self.DoB])
            + ")"
        )

    def __repr__(self) -> str:
        return (
            "("
            + ", ".join([self.ID_Student, self.Student_Name, self.ID_Class, self.DoB])
            + ")"
        )

    def __hash__(self) -> int:
        return hash(self.ID_Student)
